sdk/cosmos/azure-cosmos/azure/cosmos/http_constants.py

```diff
@@ -269,9 +269,18 @@ class CookieHeaders(object):
 class Versions(object):
     """Constants of versions.
     """
-    CurrentVersion = "2018-12-31"
-    SDKName = "azure-cosmos"
-    QueryVersion = "1.0"
+    current_version = "2018-12-31"
+    SDK_name = "azure-cosmos"
+    query_version = "1.0"
+    api_versions = [
+        "2018-06-18",  # multi-master and partition migration
+        "2018-08-31",  # throughput-split
+        "2018-09-17",  # IndexV2/ updated policy (composite and additional spatial indexes)
+        "2018-12-31",  # non-partitioned collection deprecation and migration of existing collections
+        "2019-10-14",  # Merge support in SDK
```

Comment:
What is the point having these versions when the SDK does not support them?

Reply:
@j82w Thinking from the point of view of someone who joined the SDK without much guidance - if in the future we want to see which api versions support different changes (specially for future versions, once the SDK is merge proof for instance), having the versions handy within the SDK seems like a good move (rather than going to the .NET SDK to check these).

The older versions I have no issues with removing, but the newer ones I would argue for keeping for the case I outlined before, and regardless we're not giving users a way to use these versions.

I guess the question would be if there is some public/ microsoft internal document where a new Cosmos developer would be able to find these, and if so then this would not be needed.

Reply:
Removed this to move forward, will document later on if needed.